numbers.txt

```diff
@@ -1,6 +1,7 @@
+0
 1
 2
 3
 4
 5
-6
+6
```

Comment:
Why did you delete my line??????????????????

Reply:
i did not 
i just inserted the 0 above all the lines